source/uwp/Renderer/lib/AdaptiveMediaEventInvoker.cpp

```diff
@@ -8,11 +8,6 @@
 
 namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
 {
-    AdaptiveMediaEventInvoker::AdaptiveMediaEventInvoker(winrt::RenderedAdaptiveCard const& renderResult) :
-        m_weakRenderResult(renderResult)
-    {
-    }
-
     void AdaptiveMediaEventInvoker::SendMediaClickedEvent(winrt::AdaptiveMedia const& mediaElement)
     {
         if (auto renderResult = peek_innards<implementation::RenderedAdaptiveCard>(m_weakRenderResult.get()))
```

Comment:
Likely m_weakRenderResult will never be assigned and are dead code

Reply:
It is assigned in **AdaptiveRenderContext**:
![image](https://user-images.githubusercontent.com/21356912/152241405-2d38d80c-026d-447e-99d2-6a6cf62d3eed.png)

Reply:
And if you're talking that it won't be assigned due to lack of ctor, we have a ctor in the **.h** file:
![image](https://user-images.githubusercontent.com/21356912/152241860-60e71b94-364e-4ec5-9067-79e858d1f0e1.png)